config/initializers/features.rb

```diff
@@ -1,5 +1,3 @@
-unless Rails.env.test?
-  [:feat__apple_login].each do |feat_name|
-    Flipper.enable(feat_name) unless Flipper.exist?(feat_name)
-  end
-end
+# [:feat__apple_login].each do |feat_name|
```

Comment:
I recall this causing issues with specs at the point when the table didn't exist - hence the condition on it. I'm curious if this fixes the issue in production 🤞🏾